sql/logic_test.go

```diff
@@ -401,7 +402,7 @@ SET DATABASE = test;
 }
 
 func (t *logicTest) execStatement(stmt logicStatement) {
-	if testing.Verbose() {
+	if testing.Verbose() || log.V(1) {
```

Comment:
Do we need to look at both? I think I would just use `log.V()`. `testing.Verbose()` is kind of weird; it means several things including "print output even on success", so we turn it on in our CI builds whether we want to increase the verbosity of the logs or not.